Simulator/simulator/plant_presets.py

```diff
@@ -64,58 +70,149 @@ def _compute_from_table_values(
-                                           seed_spacing=4, maturation_time=(68, 1), stopping_color=(181/255, 134/255, 1),
-                                           color_step=(10/255, -10/255, 0/255)),
-    # "dill": _compute_from_table_values(name="dill", color=(79/255, 151/255, 66/255), germination_time=(8.5, 1),
-    #                                    seed_spacing=13.5, maturation_time=(70, 1), stopping_color=(189/255, 0, 1)),
-    "fennel": _compute_from_table_values(name="fennel", color=(167/255, 247/255, 77/255), germination_time=(10, 1),
-                                         seed_spacing=11, maturation_time=(65, 1), stopping_color=(127/255, 87/255, 1),
-                                         color_step=(-5/255, -20/255, 0/255)),
-    "marjoram": _compute_from_table_values(name="marjoram", color=(101/255, 179/255, 53/255), germination_time=(10.5, 1),
-                                           seed_spacing=8, maturation_time=(60, 1), stopping_color=(181/255, 99/255, 1),
-                                           color_step=(10/255, -10/255, 0/255)),
-    "oregano": _compute_from_table_values(name="oregano", color=(147/255, 199/255, 109/255), germination_time=(11, 1),
-                                          seed_spacing=13.5, maturation_time=(88, 1), stopping_color=(122/255, 99/255, 1),
-                                          color_step=(-5/255, -10/255, 0/255)),
-    "tarragon": _compute_from_table_values(name="tarragon", color=(117/255, 158/255, 81/255), germination_time=(10.5, 1),
-                                           seed_spacing=21, maturation_time=(60, 1), stopping_color=(152/255, 88/255, 1),
-                                           color_step=(5/255, -10/255, 0/255)),
-    "nastursium": _compute_from_table_values(name="nastursium", color=(142/255, 199/255, 52/255),
-                                             germination_time=(11, 1),
-                                             seed_spacing=11, maturation_time=(60, 1), stopping_color=(202/255, 129/255, 1),
-                                             color_step=(10/255, -10/255, 0/255)),
-    "marigold": _compute_from_table_values(name="marigold", color=(117/255, 128/255, 81/255), germination_time=(7.5, 1),
-                                           seed_spacing=7, maturation_time=(50, 1), stopping_color=(177/255, 98/255, 1),
-                                           color_step=(10/255, -5/255, 0/255)),
-    # "calendula": _compute_from_table_values(name="calendula", color=(62/255, 129/255, 78/255), germination_time=(8.5, 1),
-    #                                         seed_spacing=12, maturation_time=(50, 1), stopping_color=(182/255, 129/255, 1)),
+    # https://www.gardeningknowhow.com/edible/herbs/borage/borage-herb.htm
+    "borage": _compute_from_table_values(name="borage", color=[(9 / 255, 77 / 255, 10 / 255),(0.9467, 0.6863, 0.2431)][SEG_COLORS], germination_time=(8, 5),
```

Comment:
Just checking that you guys want std_dev 5 and 1 for all germination and maturation times respectively correct?

Reply:
@mpresten is this correct?

Reply:
@wiesbadener I'm adjusting it to be plant specific for germ_time. We could just have it be 5 for all maturation_times though